User option to prepare container after creation for volume copy-up. (#676)
containers/common: docs/containers.conf.5.md

Maximum number of processes allowed in a container. 0 indicates that no limit
is imposed.

**prepare_on_create**=false

> Prepare the container on creation for volume copy-up.

Reviewer: How about something like:
```
Copy up the content from underlying image when a container with builtin volumes is created into the newly created volume. When false, the content will not be copied up until the container is started.
```

Reviewer: Once this is agreed upon, then copy it into all three places.

Reviewer: @mheon @TomSweeneyRedHat PTAL

Author: I will wait for @mheon or @TomSweeneyRedHat green signal before I update the description everywhere.

> Copy up the content from underlying image into the newly created volume when the container is created as opposed to when it is started. If `false`, the container engine will not copy up the content until the container is started. Setting it to `true` may have negative performance implications.

Reviewer 2: ```suggestion
Copy the content from the underlying image into the newly created volume when the container is created instead of when it is started. If `false`, the container engine will not copy the content until the container is started. Setting it to `true` may have negative performance implications.
```
